### PR TITLE
Fix voice focus integration/canary test

### PR DESCRIPTION
**Pull request by anuranduttaroy**
**Issue #:** NA

**Description of changes:** Fix voice focus canary/integration test to use `&m=<meeting-id>` instead of `?m=<meeting-id>` as there are already multiple other query params appended to the demo app URL for the voice focus tests.

**Testing**

1. Have you successfully run `npm run build:release` locally?
Yes.

2. How did you test these changes?
Ran the integration tests for voice focus and content share locally and verified the tests succeed.

3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it?
NA. These are integration test changes. Can be verified by checking the results of integration/canary tests.

4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
NA

5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
NA

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   `setSinkId`, and the demo will not log an error in these cases.
 - [Demo] The meeting readiness checker no longer re-initializes the device output list
   after the user picked a device.
+- [Test] Fix voice focus integration/canary test.
```
Comment (ltrung) [inline CHANGELOG.md:23]: We dont need to do CHANGELOG for Test and Demo anymore.

File: integration/js/utils/SdkBaseTest.js [hunk 2]
```diff
@@ -103,7 +103,7 @@ class SdkBaseTest extends KiteBaseTest {
       this.io.emit("setup_test", this.baseUrl, this.attendeeId);
     } else {
       this.meetingTitle = uuidv4();
-      if (this.testName === 'ContentShareOnlyAllowTwoTest') {
+      if (this.testName === 'ContentShareOnlyAllowTwoTest' || this.testName === 'Voice Focus') {
```
Comment (ltrung) [inline integration/js/utils/SdkBaseTest.js:106]: Where is the URL param for Voice Focus?

Comment (anuranduttaroy) [reply integration/js/utils/SdkBaseTest.js:106]: Synced with you offline.

File: CHANGELOG.md (hunk 1 above)
Comment (anuranduttaroy) [reply CHANGELOG.md:23]: It's not required. But isn't it good to have?

**Review by ltrung — APPROVED** [review]

**Review by ltrung — APPROVED** [review]

**Pull request merged**